### DISCUSSION TITLE
'cy.click' or 'click({ force: true })' or 'trigger('mouseover').click({ force: true })' sometimes doesnt work when running on azure pipeline or local

### DISCUSSION THREAD

**Discussion by minhngocphung** [cypress-io/cypress#27428] [CI setup]
### Current behavior

I tried all these 'cy.click' or 'click({ force: true })' or 'trigger('mouseover').click({ force: true })' , but most of the time it fails when running scripts on azure pipeline or even local (headless and non-headless)

### Desired behavior

Cypress should aways click on the element but it doesn't

### Test code to reproduce

cy.get('a[data-cy="ChangeLocationLink"]).click()

### Cypress Version

12.13.0

### Node version

18.16.0

### Operating System

windows 11 Enterprise 10.0.22...

### Other

it stops my regression test scripts running daily on production, please help urgently

**Comment by marktnoonan** [comment]
Thanks for reporting this issue. There's not enough information provided to determine whether this is an issue with Cypress. I'm going to convert this to a discussion for now.

To get help in the discussion, please add details like

- what happens when it fails
- what is the failure message, what error do you get
- what is the DOM in the part of the page being tested

Depending on the failure message and the application being tested, there could be many explanations.
